conans/client/conan_api.py

```diff
@@ -264,7 +264,8 @@ def new(self, name, header=False, pure_c=False, test=False, exports_sources=Fals
                         circleci_gcc_versions=circleci_gcc_versions,
                         circleci_clang_versions=circleci_clang_versions,
                         circleci_osx_versions=circleci_osx_versions,
-                        template=template, cache=self.app.cache)
+                        template=template, template_dir=template_dir, 
```

Comment:
Shouldn't the template be a path, it can be a path to a file or a path to a folder? Actually it seems the arg is not used in the cmd new.py

Reply:
Fixed